pkg/errox/utils.go

```diff
@@ -14,3 +20,41 @@ func IsAny(err error, targets ...error) bool {
 	}
 	return false
 }
+
+func x[T error](err error) T {
```

Comment:
Where  it's used?
Also IsAny can be deleted maybe not in this PR

Reply:
Oops, a leftover.